rllib/core/models/tests/test_mlp_heads.py

```diff
@@ -16,12 +16,13 @@ def test_mlp_heads(self):
         # Loop through different combinations of hyperparameters.
         inputs_dims_configs = [[1], [50]]
         list_of_hidden_layer_dims = [[], [1], [64, 64], [512, 512]]
-        hidden_layer_activations = ["linear", "relu", "tanh", "swish"]
```

Comment:
Just out of curiosity, why are we removing tanh?

Reply:
Simply to make the test smaller. It got larger b/c we now have different bias settings for hidden and output.